lib/connect.js

```diff
@@ -443,7 +443,7 @@ var KiteConnect = function(params) {
      * @instance
      */
     self.getLoginURL = function() {
-        return self.default_login_uri + "?api_key=" + self.api_key + "&v=3";
+        return self.default_login_uri + "?api_key=" + self.api_key + "&v=" + kiteVersion;
```

Comment:
Is this `kiteVersion` var defined anywhere? I don't see it. It's actually not required now.

Reply:
My bad, I was checking wrong branch locally and missed this.